list.c

```diff
@@ -0,0 +1,99 @@
+#include "list.h"
+
+node CreateNode(int value){
+    node temp;
+    temp = (node)malloc(sizeof(struct LinkedList));
+    temp->next = NULL;
+    temp->data = value;
+    return temp;
+}
+
+node AddTail(node head, int value){
+    node temp,p;
+    temp = CreateNode(value);
+    if(head == NULL){
+        head = temp;
+    }
+    else{
+        p  = head;
+        while(p->next != NULL){
+            p = p->next;
+        }
+        p->next = temp;
+    }
+    return head;
+}
+
+void Traverser(node head){
+    for(node p = head; p != NULL; p = p->next){
+        printf("%3d", p->data);
+    }
+}
+
+node InitHead(){
+    node head;
+    head = NULL;
```

Comment:
Should it be head->next = NULL instead head = NULL?

Reply:
hàm khởi tạo node head á anh, không có phần tử nào nên head = NULL